.github/workflows/test_on_pg9.5-3.0.yml

```diff
@@ -0,0 +1,52 @@
+name: test_on_pg9.5-3.0
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        dotnet-version: ['2.1.x' ]
+    services:
+      postgis:
+        image: postgis/postgis:9.5-3.0-alpine
+        env:
+          POSTGRES_USER: reader
+          POSTGRES_PASSWORD: reader
+          POSTGRES_DB: postgistest
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+          
+    steps:
+    - name: init database table owm_cities
+      run: psql -h localhost -p 5432 -d postgistest -U reader -c "CREATE TABLE IF NOT EXISTS owm_cities(  id   integer primary key,  city_name  varchar not null,  geom       geometry not null)"
+      env:
+        PGPASSWORD: reader
+
+    - name: init database table owm_cities
```

Comment:
Same name as on previous step?

Reply:
Sorry.Fix this later

Reply:
Hey @apdevelop  ,What's the major pg/postgis versions tests should run on ?

Reply:
I currently use only PostgreSQL 9.6.1 with PostGIS 3.0

Reply:
With github workflow matrix,i build a test workflow [here](https://github.com/sharkAndshark/linq2db-postgis-extensions/tree/runtest2/.github/workflows),it can run tests on various versions of PostgreSQL/PostGIS .
But getting this work seems need group tests,because some method only support PostGIS.version >= 2.5 or 3.0 or 2.5,etc.

Reply:
I never use such tools... can you briefly explain (or give a link), how these `.yml` files will be used?

Reply:
With .yml files in .github/workflows,github will run jobs which defined on the .yml files automatically trigged by PR/push.
I defined jobs in these .yml files  which will start various versions postgis,and run tests on these postgis.

Check the [github workflow doc](https://docs.github.com/en/actions) here,and this [PR's results](https://github.com/sharkAndshark/linq2db-postgis-extensions/actions) here.
I'm new to this tool also; 
Expecting your view .

Reply:
I merged it, but there are seems to be some syntax errors in `.yml` files